sematic/ui/packages/common/src/pages/RunDetails/artifacts/artifact.tsx

```diff
@@ -54,11 +72,19 @@ function Artifact(props: ArtifactProps) {
     return <>
         <StyledSection>
             <Headline>Artifact</Headline>
+            <ControlButtonsGroup>
+                <Button variant={"text"} disabled={coordinationState === ArtifactCoordinationState.EXPAND_ALL}
+                    onClick={() => setCoordinationState(ArtifactCoordinationState.EXPAND_ALL)} >Expand all</Button>
+                <Button variant={"text"} disabled={coordinationState === ArtifactCoordinationState.COLLAPSE_ALL} 
+                    onClick={() => setCoordinationState( ArtifactCoordinationState.COLLAPSE_ALL)} >Collapse all</Button>
+            </ControlButtonsGroup>
```

Comment:
These actions should be in the context menu that opens when clicking the ellipsis.

Reply:
Cool. Made it that way.